annotate.nf

```diff
@@ -73,23 +73,23 @@ vcfToAnnotate = Channel.create()
 vcfNotToAnnotate = Channel.create()
 
 if (annotateVCF == []) {
+// by default we annotate both germline and somatic results that we can find in the VariantCalling directory
```

Comment:
In fact, we annote all available vcfs by default, so it's not really a question of germline/somatic, but really more a question of which tools was run